README.md

```diff
@@ -55,12 +55,12 @@ The fourth parameter is an `$options` array. The additional options are:
 For example, by default calls will be made over a non-encrypted connection. To change this to make calls over HTTPS use:
 
 ```php
-$pusher = new Pusher( $app_key, $app_secret, $app_id, array( 'cluster' => $app_cluster, 'encrypted' => true ) );
+$pusher = new Pusher\Pusher( $app_key, $app_secret, $app_id, array( 'cluster' => $app_cluster, 'encrypted' => true ) );
```

Comment:
I'm wondering how to update documentation... Should we suggest `new Pusher\Pusher` or should we say `use Pusher` to import this namespace and keep suggesting `new Pusher` here? (I'd appreciate your input on this as well @leesio).